### ISSUE TITLE
FN training

### ISSUE THREAD

**Issue by zhly0** [KaleidoZhouYN/Details-on-Face-Recognition#2]
thanks for your code.
In the deploy.prototxt,the last batch_norm layer is 
layer {
  name: "fc5_bn"
  type: "BatchNorm"
  bottom: "fc5"
  top: "fc5"
    param {
    lr_mult: 0
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
  param {
    lr_mult: 0
    decay_mult: 0
  }
}
what should I do when training,any parameter that need modify?or just putting the FN layer after the feature layer without any change?

**Comment by KaleidoZhouYN** [comment]
don't need to do any modification when training.
you can also apply a scale layer to modify the scale parameter if needed.

**Comment by zhly0** [comment]
thanks,got it!
